src/components/Hints/__snapshots__/index.test.js.snap

```diff
@@ -1,3 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Hints should render jsx 1`] = `null`;
+
```

Comment:
I don't think this snapshot is very useful for this specific case.

Reply:
Okay